sleap/gui/dataviews.py

```diff
@@ -683,6 +683,40 @@ def item_to_data(self, obj: RecordingSession, item: Camcorder):
         video = obj.get_video(item)
         return {"camera": item.name, "video": video.filename if video else ""}
 
+class CameraGroupsTableModel(GenericTableModel):
+    """Table model for camera groups."""
+    
+    properties = ("name", "cameras")
+    
+    def __init__(self, items=None, context=None):
+        super().__init__(items=items, context=context)
```

Comment:
Expose the property kwarg:
```suggestion
    def __init__(self, items=None, properties=None, context=None):
        super().__init__(items=items, properties=properties, context=context)
```